src/circuit.ts

```diff
@@ -33,6 +33,15 @@ export class Circuit {
             ...this._circuitConfig.compileOptions
         })
 
+       
+    }
+
+    async compile() {
+        await this._compile()
+  
+        if(!this._circuitConfig.powerOfTauFp || this._circuitConfig.powerOfTauFp.length < 1){
```

Comment:
`this._circuitConfig.powerOfTauFp. === ""` is a better condition, make clear what the condition is about.